nucypher/utilities/prometheus/collector.py

```diff
@@ -107,6 +107,9 @@ def initialize(self, metrics_prefix: str, registry: CollectorRegistry) -> None:
             "policies_held_gauge": Gauge(f'{metrics_prefix}_policies_held',
                                          'Policies held',
                                          registry=registry),
+           "availability_score_gauge": Gauge(f'{metrics_prefix}_availability_score',
+                                         'Availability score',
+                                         registry=registry),
```

Comment:
Text alignment could be improved here:

```suggestion
            "availability_score_gauge": Gauge(f'{metrics_prefix}_availability_score',
                                                                      'Availability score',
                                                                      registry=registry),
```